docs/run-preinstalled-wda.md

```diff
@@ -114,3 +114,12 @@ driver = @core.start_driver
 # do something
 driver.quit
 ```
+
+> **Note**
+> As of iOS 17, the testmanagerd service name has changed to `com.apple.dt.testmanagerd.runner` from `com.apple.testmanagerd`.
+> It causes an unexpected WDA process crash with embedded XCTest frameworks to run a single WebDriverAgent package to various OS environments without `xcodebuild`.
```

Comment:
It causes an WDA process crash with embedded XCTest frameworks to run a single WebDriverAgent package to various OS environments without `xcodebuild` -> ... while running a single WebDriverAgent package on various OS environments without `xcodebuild`